contrib/scripts/custom-vet-check.sh

```diff
@@ -6,4 +6,4 @@
 # https://github.com/cilium/customvet
 # It performs custom static analysis checks checks for the
 # cilium repository.
-go run github.com/cilium/customvet -timeafter.ignore inctimer -readall.ignore safeio ./...
+${GO} run github.com/cilium/customvet -timeafter.ignore inctimer -readall.ignore safeio ./...
```

Comment:
```suggestion
"$GO" run github.com/cilium/customvet -timeafter.ignore inctimer -readall.ignore safeio ./...
```

* to handle the case if someone uses a crazy path with spaces.
* to prevent argument injection via $GO.